edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/ConnectionProvider.cs

```diff
@@ -1,15 +1,20 @@
 // Copyright (c) Microsoft. All rights reserved.
-
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
 namespace Microsoft.Azure.Devices.Edge.Hub.Core
 {
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
     using System.Threading.Tasks;
+
     using Microsoft.Azure.Devices.Edge.Hub.Core.Device;
     using Microsoft.Azure.Devices.Edge.Hub.Core.Identity;
     using Microsoft.Azure.Devices.Edge.Util;
 
     public class ConnectionProvider : IConnectionProvider
     {
         readonly IConnectionManager connectionManager;
+
```

Comment:
Is there a rule that requires a newline between these member declarations? I kind of like the current style with the bunched declarations. C# already uses enough vertical space 😃

Reply:
Yes.  I will remove this format.